RestaurantReservation.Db/Repositories/EmployeesRepository.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Repositories
+{
+    public class EmployeesRepository
+    {
+        private readonly RestaurantReservationDbContext _context;
+        public EmployeesRepository(RestaurantReservationDbContext context)
+        {
+            _context = context;
+        }
+        public async Task<int> CreateEmployee(Employees newEmployee)
+        {
+            await _context.Employees.AddAsync(newEmployee);
+            await _context.SaveChangesAsync();
+            return newEmployee.Id;
+        }
+        public async Task<Employees> GetEmployee(int employeeId)
+        {
+            return await _context.Employees.FindAsync(employeeId);
+        }
+        public async Task DeleteEmployee(int employeeId)
+        {
+            var employee = await GetEmployee(employeeId);
+            _context.Employees.Remove(employee);
+            await _context.SaveChangesAsync();
+        }
+        public async Task UpdateEmployee(Employees updateEmployee)
+        {
+            _context.Employees.Update(updateEmployee);
+            await _context.SaveChangesAsync();
+        }
+        public async Task<List<Employees>> ListManagers()
+        {
+            var position = "Manager";
```

Comment:
> We try to avoid to have hardcoded strings in code, if you want to change the name of it, will be hard to do that change to all fille, and reusing it is tricky. So please move this to a constant file call EmployeePositions.

By Randa.